lib/sslcommon.c

```diff
@@ -200,8 +200,13 @@ static int gftp_ssl_post_connection_check (gftp_request * request)
               ext_str = meth->d2i(NULL, &mutable_ptr, len);
 #endif
               val = meth->i2v(meth, ext_str, NULL);
-
-              for (j = 0; j < sk_CONF_VALUE_num(val); j++)
+	/**************************************************************************
+	* 20250414: Nicolas Baranger  
+	* Adding support of TLS validation for wildcard domain names (*.domain.tld)
+	* present in certificate subjectAltName field  
+	****************************************************************************/
```

Comment:
Please drop the comments with the names / dates for consistency with the rest of the code. This information is stored in the git commit log. `git blame` is a useful tool to show exactly what you want here.